example/launcher_cr.yaml

```diff
@@ -6,8 +6,13 @@ spec:
 
 ####### OpenShift Configuration
   openshift:
-    consoleUrl: #<the OpenShift Console Url (ex: 'https://192.168.64.4:8443')>#
+    consoleUrl: https://192.168.42.39:8443/console
```

Comment:
@edewit is it supposed to be your config?

Reply:
oops